modules/budget/variables.tf

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "billing_account" {
+  description = "ID of the billing account to set a budget on"
+  type        = string
+}
+
+variable "projects" {
+  description = "The project ids to include in this budget. If empty budget will include all projects"
+  type        = list(string)
+}
+
+variable "amount" {
+  description = "The amount to use as the budget"
+  type        = string
+}
+
+variable "create_budget" {
+  description = "If the budget should be created"
+  type        = bool
+  default     = true
+}
+
+variable "display_name" {
+  description = "The display name of the budget"
```

Comment:
Please add default behavior if unset to description.

Reply:
Done